File: Feitos em Java/2006.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner(System.in);
        int cha=entrada.nextInt();
        int vet[]=new int [5];
        for(int x=0;x<5;x++){
            vet[x]=entrada.nextInt();
        }
        int vezes=0;
        for(int x=0;x<5;x++){
            if(vet[x]==cha){
                vezes++;
            }
        }
        System.out.println(vezes);
    }
}

File: Feitos em C++/2006.cpp
#include <iostream>
#include<stdio.h>
using namespace std;
int main(){
    int cha,vet[5];
    cin>>cha;
    for(int x=0;x<5;x++){
        cin>>vet[x];
    }
    int vezes=0;
    for(int x=0;x<5;x++){
        if(vet[x]==cha){
            vezes++;
        }
    }
    cout<<vezes<<endl;
    return 0;
}
